src/main/java/com/github/manolo8/darkbot/core/objects/facades/SeassonPassMediator.java

```diff
@@ -0,0 +1,135 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import com.github.manolo8.darkbot.core.objects.facades.QuestProxy.Quest;
+import com.github.manolo8.darkbot.core.objects.swf.FlashList;
+
+import eu.darkbot.api.managers.QuestAPI;
+import eu.darkbot.api.managers.SeassonPassAPI;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.ToString;
+
+import static com.github.manolo8.darkbot.Main.API;
+
+import java.util.List;
+
+import org.jetbrains.annotations.Nullable;
+
+public class SeassonPassMediator extends Updatable implements SeassonPassAPI {
+
+    @Getter
+    private CurrentLevelProgress currentLevelProgress = new CurrentLevelProgress();
+
+    @Getter
+    private CurrentLevelProgress currentSeassonLevelInfo = new CurrentLevelProgress();

```

Comment:
These two can be final, idea would've warned about this too:
![image](https://github.com/darkbot-reloaded/DarkBot/assets/11789291/c060043d-26b7-46ce-a641-399179af0be6)

Reply:
done